### PR TITLE
Update plots and dilepton unfolding

### PR DESCRIPTION
**Pull request by davidwalter2**
- On top of #546 
- Temporary fix for unfolding where the order of datasets in the histmaker is inverted
- Update plots that include unfolded data

### CODE REVIEW

**Comment by davidwalter2** [comment]
OK, I think this is now good to be reviewed/ merged.
PRValidation shows no difference in the results:
this PR: https://cmsmwbot.web.cern.ch/WMassAnalysis/PRValidation/PR542/2024_09_28/
reference: https://cmsmwbot.web.cern.ch/WMassAnalysis/PRValidation/ReferenceRuns/2024_09_27_a76ad91/

File: scripts/histmakers/w_z_gen_dists.py [hunk 1]
```diff
@@ -93,8 +93,15 @@ def build_graph(df, dataset):
             common.get_gen_axes(common.get_dilepton_ptV_binning(), True, flow=True), 
             add_out_of_acceptance_axis=False,
         )
-        axis_ptVgen = unfolding_axes[0]
-        axis_absYVgen = unfolding_axes[1]
+        axis_absYVgen = hist.axis.Variable(
+            unfolding_axes[1].edges,
+            name = "absYVgen", underflow=False, overflow=False,
```
Comment (bendavid) [inline scripts/histmakers/w_z_gen_dists.py:98]: Is it safe to have no overflow for this axis?

Comment (davidwalter2) [reply scripts/histmakers/w_z_gen_dists.py]: Fixed, I also tried to clean up the flow bins in other cases

Comment (bendavid) [reply scripts/histmakers/w_z_gen_dists.py]: Did you check if this has any impact on the problems which led to the dataset order being reversed?

Comment (davidwalter2) [reply scripts/histmakers/w_z_gen_dists.py]: I didn't check this issue again but plan to have a look soon. Do you think it should be understood before merging this PR?

Comment (bendavid) [reply scripts/histmakers/w_z_gen_dists.py]: I think it's worth at least checking if the reversal is still needed.  If yes we can still merge it as-is.

Comment (davidwalter2) [reply scripts/histmakers/w_z_gen_dists.py]: Quickly checked but it's still crashing

**Pull request merged**